mobile/src/stateMachine/views/healthData/healthScopes.tsx

```diff
@@ -0,0 +1,180 @@
+                        <Text style={[TextStyles.p1, styles.instructionText]}>{'Tap Health'}</Text>
+                    </View>
+                    <View style={styles.imageNextToText}>
+                        <Image source={Images.iOS_CompanionKit} style={{ width: 26, height: 26 }} />
+                        <Text style={[TextStyles.p1, styles.instructionText]}>{'Select CompanionKit'}</Text>
+                    </View>
+                    <View style={styles.imageNextToText}>
+                        <Image source={Images.iOS_Switch} style={{ width: 26, height: 26 }} />
+                        <Text style={[TextStyles.p1, styles.instructionText]}>{'Turn All Categories On'}</Text>
+                    </View>
+                </View>,
+            onConfirm: async () => {
+                AppController.Instance.PromptModal.closeModal();
+                // Sub-optimal ways to open iOS settings
+                // const pathParts = ['HEALTH', '&path=', 'Privacy', ':', 'prefs', 'App-']; // Private API could get rejected (obscured here)
+                // const path = pathParts.reverse().join(''); 
+                const path = 'app-settings:'; // opens CompanionKit specific settings (not health) by using Public API
+                await Links.tryOpenLink(path);
+            },
+        });
+
+    }
+    renderContent() {
+        // const enabled = Platform.OS == 'ios' ? this.model.isEnabledOG : this.model.isEnabled;
+        // const permissionsEnabled = enabled && !this.model.isToggleInProgress;
+        const titleText = 'Health Data';
+        const explaining = 'We are collecting your health data to build a better personalized experience for you in the app.';
+        const perm = this.model.getPermissions();
+        return (
+            <MasloPage style={this.baseStyles.page} onBack={() => this.trigger(ScenarioTriggers.Back)} onClose={() => this.trigger(ScenarioTriggers.Cancel)}>
+                <Container style={styles.topBarWrapWrap}>
+                    <PersonaScrollMask />
+                </Container>
+                <ScrollView style={[{ zIndex: 0, elevation: 0 }]}>
+                    <Container style={[this.baseStyles.container, styles.container]}>
+                        <Text style={[this.textStyles.h1, styles.title]}>{titleText}</Text>
+                        <Text style={[this.textStyles.p1, styles.subTitle]}>{explaining}</Text>
+                        {/* <Card
+                            title={`${Platform.OS == 'ios' ? 'HealthKit' : 'GoogleFit'} Data`}
+                            description={'Change Authorization'}
+                            style={{ marginBottom: 20 }}
+                        >
+                        </Card> */}
+                        {Platform.OS == 'ios' && (
+                            <View style={styles.buttonView}>
+                                <Button
+                                    title='Grant Permission'
```

Comment:
on the settings page - should we change "grant permissions" to be "change permissions" or "check permissions" or otherwise? we do not have access to read permissions from healthkit so we do not know the exact permissions state